src/k2/transport/RetryStrategy.h

```diff
@@ -100,7 +100,7 @@ class ExponentialBackoffStrategy {
                         K2LOG_D(log::txretry, "round {} ended with success={}", _try, _success);
                         if (!_success) {
                             _try++;
-                            _currentTimeout = Duration((uint64_t)(nsec(_currentTimeout).count() * _rate));
+                            _currentBackoffTime = Duration((uint64_t)(nsec(_currentBackoffTime).count() * _rate));
```

Comment:
I think can also change to 
```suggestion
                            _currentBackoffTime = _currentBackoffTime * _rate;
```

Reply:
This will trigger "no match for operator=" error. 
"_currentBackoffTime *= _rate" works fine.